client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	require( 'test/helpers/use-fake-dom' )();
+
+	describe( 'component rendering', function() {
+		it( 'should render a list with the provided options', function() {
+			const dropdownOptions = getDropdownOptions();
+			const dropdown = mount( <SelectDropdown options={ dropdownOptions } /> );
+
+			assert.equal( 'Statuses', dropdown.find( '.select-dropdown__label' ).text() );
```

Comment:
I agree that for specific assertions `expect` may lead to a more readable code.
At work we use QUnit, for personal projects I usually go for tape... so the `assert` style comes more natural to me.

I guess this could be the occasion to get more comfortable with `expect`.